src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import static java.lang.Math.sin;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int radiusInscribedCircle;
+    private int angleBase;
+
+    public IsoscelesTrapezoid(String color, int radiusInscribedCircle, int angleBase) {
+        this.radiusInscribedCircle = radiusInscribedCircle;
+        this.color = color;
```

Comment:
since color is common, would be good to have super(color) - constructor for color in parent class Figure

Reply:
not fixed